packages/element-templates-json-schema-shared/src/defs/condition.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$id": "#/condition",
+  "type": "object",
+  "description": "Condition to activate the binding.",
+  "allOf": [
+    {
+      "$ref": "examples.json#/condition"
+    }
+  ],
+  "required": [
+    "property"
+  ],
+  "errorMessage": {
+    "required": {
+      "property": "missing property name for condition \"${0#}\""
+    }
+  },
+  "properties": {
+    "type": {
+      "$id": "#/condition/type",
+      "const": "simple",
```

Comment:
If we are that explicit, should we have a test for this? `type !== simple`.

Reply:
I will add this.